### ISSUE TITLE
Library and asset download prompt should be more detailed as well as output details to the console log

### ISSUE THREAD

**Issue by TheMCNerd2017** [MultiMC/Launcher#4329]
### Role

Playing multiple different versions of Minecraft and having to wait for libraries and assets to be redownloaded.

### Suggestion

The library and asset update prompt should be more detailed, as well as output download information to the console log.

### Benefit

It would be useful to know which libraries and assets are being downloaded as well as their progress.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

Basically I'm suggesting that both the library and asset download prompt should be changed to show the current file being downloaded, the progress of the current file and for the information to be outputted to the console log.
